sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/stringExpressions.scala

```diff
@@ -2538,6 +2538,76 @@ case class Encode(value: Expression, charset: Expression)
     newLeft: Expression, newRight: Expression): Encode = copy(value = newLeft, charset = newRight)
 }
 
+/**
+ * Converts the input expression to a binary value based on the supplied format.
+ */
+// scalastyle:off line.size.limit
+@ExpressionDescription(
+  usage = """
+    _FUNC_(str[, fmt]) - Converts the input `str` to a binary value based on the supplied `fmt`.
+      By default, the binary format for conversion is "hex" if `fmt` is omitted.
```

Comment:
How do you think setting `hex` as the default format? There is another [reference](https://docs.snowflake.com/en/sql-reference/functions/to_binary.html) setting `hex` as the default format.